app/src/main/java/net/squanchy/imageloader/ImageLoaderComponent.java

```diff
@@ -0,0 +1,12 @@
+package net.squanchy.imageloader;
+
+import net.squanchy.injection.ActivityLifecycle;
+
+import dagger.Component;
+
+@ActivityLifecycle
```

Comment:
are we sure we want this to be activity specific? I would see this as a component shared across the app. I'm ok to move it when we use it in another activity tho.